repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -294,7 +295,7 @@ def report_skipped_packages(message, packages):
         reporting.Summary(summary),
         reporting.Severity(reporting.Severity.HIGH),
         reporting.Tags([reporting.Tags.REPOSITORY]),
-    ])
+    ] + [reporting.RelatedResource('package', p) for p in packages])
```

Comment:
TypeError: Values "scheme" and "identifier" of "RelatedResource" must be a string